minio/policy.py

```diff
@@ -66,7 +81,7 @@ def _new_bucket_statement(policy, bucket_name, prefix=''):
     if policy == Policy.NONE:
         return []
 
-    bucket_resource = _AWS_RESOURCE_PREFIX + bucket_name
+    bucket_resource = [_AWS_RESOURCE_PREFIX + bucket_name]
```

Comment:
I thought resource has to be a list. Didn't know, it can either be list or string.